builtin/providers/aws/resource_aws_ses_active_receipt_rule_set.go

```diff
@@ -0,0 +1,82 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsSesActiveReceiptRuleSet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSesActiveReceiptRuleSetCreate,
+		Update: resourceAwsSesActiveReceiptRuleSetUpdate,
+		Read:   resourceAwsSesActiveReceiptRuleSetRead,
+		Delete: resourceAwsSesActiveReceiptRuleSetDelete,
+
+		Schema: map[string]*schema.Schema{
+			"rule_set_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceAwsSesActiveReceiptRuleSetCreate(d *schema.ResourceData, meta interface{}) error {
+	return resourceAwsSesActiveReceiptRuleSetUpdate(d, meta)
+}
+
+func resourceAwsSesActiveReceiptRuleSetUpdate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).sesConn
+
+	ruleSetName := d.Get("rule_set_name").(string)
+
+	createOpts := &ses.SetActiveReceiptRuleSetInput{
+		RuleSetName: aws.String(ruleSetName),
+	}
+
+	_, err := conn.SetActiveReceiptRuleSet(createOpts)
+	if err != nil {
+		return fmt.Errorf("Error setting active SES rule set: %s", err)
+	}
+
+	d.SetId(ruleSetName)
+
+	return resourceAwsSesActiveReceiptRuleSetRead(d, meta)
+}
+
+func resourceAwsSesActiveReceiptRuleSetRead(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).sesConn
+
+	describeOpts := &ses.DescribeActiveReceiptRuleSetInput{}
+
+	response, err := conn.DescribeActiveReceiptRuleSet(describeOpts)
+	if err != nil {
+		return err
```

Comment:
I think we need to handle the case here that Rules can be deleted from the console. If that happens, this will not refresh the rule, it will error out. An example can be found here: https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_app_cookie_stickiness_policy.go#L103

To implement this, the correct error code would need to be added